Update spanish.xml v 8.4.7

P0rsche-911 opened the pull request:
Update prior to the final update. Once approved it will be adapted to the current english.xml version to facilitate future translations.

Actualización previa a la actualización definitiva. Una vez aprobada será adaptada a la versión english.xml actual para facilitar futuras traducciones.

P0rsche-911 commented on PowerEditor/installer/nativeLang/spanish.xml:
```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
```
Unedited space. Exists in the english.xml version.

rdipardo commented on PowerEditor/installer/nativeLang/spanish.xml:
```diff
@@ -18,39 +18,39 @@ The comments are here for explanation, it's not necessary to translate them.
-					<Item subMenuId="search-copyStyledText" name="Copiar texto con estilo"/>
-					<Item subMenuId="search-bookmark" name="Marca"/>
+					<Item subMenuId="edit-copyToClipboard" name="C&amp;opiar al portapapeles"/>
+					<Item subMenuId="edit-indent" name="Tabulac&ión"/>
```
@P0rsche-911, mira la línea treinta y uno:

    <Item subMenuId="edit-indent" name="Tabulac&ión"/>

El [mensaje de error][0] significa que el `&` marca el comienzo de una entidad.
La sintaxis correcta es así: `Tabulac&amp;ión`

[0]: https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/45501747/job/j98jgkhl5wht394o/messages

conky77 commented:
@P0rsche-911 you should add a new commit with the correction (edit the previous Pull request and add a new commit)

P0rsche-911 commented:
I added a comment when editing line 1 but it doesn't show up and I don't know how to edit it either.
I have removed the trailing space that exists in the latest english.xml version but not in the spanish.xml v 8.3.3 version.
Sorry for all the errors.

Añadí un comentario al editar la línea 1 pero no se muestra y tampoco sé cómo editarlo.
He eliminado el espacio final que existe en la última versión english.xml pero no en la versión spanish.xml v 8.3.3.
Perdón por todos los errores.

P0rsche-911 reviewed (commented):
Deleted. Sorry

donho commented:
Is any Spanish speaker here can validate the modification?
